config_example.sh

```diff
@@ -683,3 +685,9 @@ set -x
 # deployment systemd services of the Agent based installation. This is
 # particularly useful for WebUI development.
 # export AGENT_DISABLE_AUTOMATED=false
+
+# When set, the code internally sets the boot mode for the agents.
+# This config variable is used only by the agent based installer and is optional.
+# The default value for AGENT_E2E_TEST_BOOT_MODE is 'ISO'.
+# When set to 'PXE', the AGENT_E2E_TEST_SCENARIO can only be set to IPV4 scenarios 
```

Comment:
I'm curious, why does IPv6 and dual stack not work?

Reply:
The limitation is reported by [libvirt documentation](https://libvirt.org/formatnetwork.html) (see `bootp` and `tftp` elements). There's no explicit reasoning for that, I guess (but I could be wrong) it was due the additional work required, and the fact IPv4 was a more common scenario

Reply:
Also, PXE is nothing but the same ignition packed as in ISO but with different packaging and we have dev-scripts support for IPV6 and Dual stack for ISO ( and in CI) which means we in a way are testing the cluster creation via another way. There is no difference in the internal contents of PXE ( initd to be specific) compared to ISO, both have the same ignition data for the cluster. So given the limitation of libvirt, what @andfasano mentioned above, it is enough to just test if we can boot using PXE and IPV4. Once the installer branch opens, QE is going to manually test PXE with IPV6 in their lab to be sure. @

Reply:
Thanks. The libvirt documentation says tftp isn't supported for IPv6. It maybe worth adding that in the comments in case someone else asks in the future.